src/commands/build/index.ts

```diff
@@ -84,6 +86,22 @@ export default class BuildCommand extends Command {
       releasePkg.typings = 'src/index.d.ts';
     }
 
+    // Convert any yalced dependencies (local "file:" dependencies) into proper bundledDependencies before publishing
+    for (const [name, version] of Object.entries(releasePkg.dependencies as { [key: string]: string })) {
+      if (version.startsWith('file:')) {
+        const filepath = version.replace('file:', '');
+        const installPath = buildPath('dist', 'node_modules', name);
```

Comment:
Is dist/node_modules actually published?

Reply:
Yup! That's how `bundledDependencies` works.

Reply:
```
npm pack
```

<details>
<summary><code>tar -tvf stoplight-studio-0.0.0.tgz</code></summary>
<code><pre>
> tar -tvf stoplight-studio-0.0.0.tgz
-rw-r--r--  0 0      0        2147 Oct 26  1985 package/package.json
-rw-r--r--  0 0      0         114 Oct 26  1985 package/App.d.ts
-rw-r--r--  0 0      0         137 Oct 26  1985 package/index.d.ts
-rw-r--r--  0 0      0      949639 Oct 26  1985 package/index.es.js
-rw-r--r--  0 0      0          13 Oct 26  1985 package/LICENSE
-rw-r--r--  0 0      0        1545 Oct 26  1985 package/README.md
-rw-r--r--  0 0      0          12 Oct 26  1985 package/root.d.ts
-rw-r--r--  0 0      0         183 Oct 26  1985 package/types.d.ts
-rw-r--r--  0 0      0         184 Oct 26  1985 package/alias/fs.d.ts
-rw-r--r--  0 0      0         622 Oct 26  1985 package/api/nodes.d.ts
-rw-r--r--  0 0      0         165 Oct 26  1985 package/components/CodeEditor/index.d.ts
-rw-r--r--  0 0      0          99 Oct 26  1985 package/components/CodePanel/index.d.ts
-rw-r--r--  0 0      0         252 Oct 26  1985 package/components/ConfirmButton/index.d.ts
-rw-r--r--  0 0      0         254 Oct 26  1985 package/components/CreateArticleForm/index.d.ts
-rw-r--r--  0 0      0         272 Oct 26  1985 package/components/CreateHttpOperationForm/index.d.ts
-rw-r--r--  0 0      0         266 Oct 26  1985 package/components/CreateHttpServiceForm/index.d.ts
-rw-r--r--  0 0      0         248 Oct 26  1985 package/components/CreateModelForm/index.d.ts
-rw-r--r--  0 0      0         210 Oct 26  1985 package/components/CreateNodeForm/form.d.ts
-rw-r--r--  0 0      0          86 Oct 26  1985 package/components/CreateNodeForm/index.d.ts
-rw-r--r--  0 0      0         105 Oct 26  1985 package/components/CreateNodeForm/modal.d.ts
-rw-r--r--  0 0      0         107 Oct 26  1985 package/components/CreateProjectForm/form.d.ts
-rw-r--r--  0 0      0          92 Oct 26  1985 package/components/CreateProjectForm/index.d.ts
-rw-r--r--  0 0      0         108 Oct 26  1985 package/components/CreateProjectForm/modal.d.ts
-rw-r--r--  0 0      0         106 Oct 26  1985 package/components/DiagnosticsPanel/index.d.ts
-rw-r--r--  0 0      0         106 Oct 26  1985 package/components/EmptyListMessage/FileTree.d.ts
-rw-r--r--  0 0      0          98 Oct 26  1985 package/components/EmptyListMessage/index.d.ts
-rw-r--r--  0 0      0         105 Oct 26  1985 package/components/EmptyListMessage/TagTree.d.ts
-rw-r--r--  0 0      0         375 Oct 26  1985 package/components/FormController/FormCodeEditor.d.ts
-rw-r--r--  0 0      0         382 Oct 26  1985 package/components/FormController/FormEditableText.d.ts
-rw-r--r--  0 0      0         561 Oct 26  1985 package/components/FormController/FormInput.d.ts
-rw-r--r--  0 0      0         326 Oct 26  1985 package/components/FormController/FormSelect.d.ts
-rw-r--r--  0 0      0         337 Oct 26  1985 package/components/FormController/FormTitle.d.ts
-rw-r--r--  0 0      0         652 Oct 26  1985 package/components/FormController/hooks.d.ts
-rw-r--r--  0 0      0        1316 Oct 26  1985 package/components/FormController/index.d.ts
-rw-r--r--  0 0      0         245 Oct 26  1985 package/components/FormHttpService/index.d.ts
-rw-r--r--  0 0      0         153 Oct 26  1985 package/components/FormJsonSchema/index.d.ts
-rw-r--r--  0 0      0         156 Oct 26  1985 package/components/FormOas2Operation/index.d.ts
-rw-r--r--  0 0      0         589 Oct 26  1985 package/components/FormOas2Operation/RequestBody.d.ts
-rw-r--r--  0 0      0         263 Oct 26  1985 package/components/FormOas3Operation/Content.d.ts
-rw-r--r--  0 0      0         156 Oct 26  1985 package/components/FormOas3Operation/index.d.ts
-rw-r--r--  0 0      0         232 Oct 26  1985 package/components/FormOasOperation/Basics.d.ts
-rw-r--r--  0 0      0         249 Oct 26  1985 package/components/FormOasOperation/Body.d.ts
-rw-r--r--  0 0      0         331 Oct 26  1985 package/components/FormOasOperation/Example.d.ts
-rw-r--r--  0 0      0         444 Oct 26  1985 package/components/FormOasOperation/Parameter.d.ts
-rw-r--r--  0 0      0         324 Oct 26  1985 package/components/FormOasOperation/RequestParameters.d.ts
-rw-r--r--  0 0      0         291 Oct 26  1985 package/components/FormOasOperation/ResponseHeaders.d.ts
-rw-r--r--  0 0      0         381 Oct 26  1985 package/components/FormOasOperation/Responses.d.ts
-rw-r--r--  0 0      0         272 Oct 26  1985 package/components/FormOasOperation/SecurityParameters.d.ts
-rw-r--r--  0 0      0         278 Oct 26  1985 package/components/FormsPanel/index.d.ts
-rw-r--r--  0 0      0         107 Oct 26  1985 package/components/GitBranchSelector/index.d.ts
-rw-r--r--  0 0      0         109 Oct 26  1985 package/components/GitCredentialDialog/GitCredentialDialog.d.ts
-rw-r--r--  0 0      0          40 Oct 26  1985 package/components/GitCredentialDialog/index.d.ts
-rw-r--r--  0 0      0         106 Oct 26  1985 package/components/HttpServiceList/index.d.ts
-rw-r--r--  0 0      0         412 Oct 26  1985 package/components/HttpServiceSelect/index.d.ts
-rw-r--r--  0 0      0         357 Oct 26  1985 package/components/JsonSchemaEditor/hooks/useRefProviders.d.ts
-rw-r--r--  0 0      0         195 Oct 26  1985 package/components/JsonSchemaEditor/hooks/useRefs.d.ts
-rw-r--r--  0 0      0         356 Oct 26  1985 package/components/JsonSchemaEditor/index.d.ts
-rw-r--r--  0 0      0         381 Oct 26  1985 package/components/ModePanel/index.d.ts
-rw-r--r--  0 0      0         379 Oct 26  1985 package/components/PanelToolbar/index.d.ts
-rw-r--r--  0 0      0         100 Oct 26  1985 package/components/PrismPanel/index.d.ts
-rw-r--r--  0 0      0          90 Oct 26  1985 package/components/ProjectActions/index.d.ts
-rw-r--r--  0 0      0         203 Oct 26  1985 package/components/ProjectSelector/index.d.ts
-rw-r--r--  0 0      0         500 Oct 26  1985 package/components/ReadPanel/components.d.ts
-rw-r--r--  0 0      0         341 Oct 26  1985 package/components/ReadPanel/handlers.d.ts
-rw-r--r--  0 0      0         279 Oct 26  1985 package/components/ReadPanel/index.d.ts
-rw-r--r--  0 0      0         246 Oct 26  1985 package/components/RemoveProjectDialog/index.d.ts
-rw-r--r--  0 0      0          99 Oct 26  1985 package/components/SearchBar/index.d.ts
-rw-r--r--  0 0      0         115 Oct 26  1985 package/components/SettingsDialog/ExperimentalFeaturesPanel.d.ts
-rw-r--r--  0 0      0         102 Oct 26  1985 package/components/SettingsDialog/GeneralPanel.d.ts
-rw-r--r--  0 0      0          98 Oct 26  1985 package/components/SettingsDialog/GitPanel.d.ts
-rw-r--r--  0 0      0         275 Oct 26  1985 package/components/SettingsDialog/index.d.ts
-rw-r--r--  0 0      0         247 Oct 26  1985 package/components/SettingsDialog/ProjectPanel.d.ts
-rw-r--r--  0 0      0          97 Oct 26  1985 package/components/SidebarTrees/GitTree/index.d.ts
-rw-r--r--  0 0      0         126 Oct 26  1985 package/components/SidebarTrees/index.d.ts
-rw-r--r--  0 0      0         168 Oct 26  1985 package/components/SidebarTrees/NodeMeta.d.ts
-rw-r--r--  0 0      0         270 Oct 26  1985 package/components/SimpleSuggest/index.d.ts
-rw-r--r--  0 0      0         184 Oct 26  1985 package/components/Studio/Fallback.d.ts
-rw-r--r--  0 0      0         196 Oct 26  1985 package/components/Studio/index.d.ts
-rw-r--r--  0 0      0         183 Oct 26  1985 package/components/Studio/Sidebar.d.ts
-rw-r--r--  0 0      0         109 Oct 26  1985 package/components/Studio/SidebarServicesList.d.ts
-rw-r--r--  0 0      0         102 Oct 26  1985 package/components/Studio/StudioLayout.d.ts
-rw-r--r--  0 0      0          98 Oct 26  1985 package/components/Studio/StudioUi.d.ts
-rw-r--r--  0 0      0         308 Oct 26  1985 package/components/TagSelect/index.d.ts
-rw-r--r--  0 0      0          96 Oct 26  1985 package/components/TopBar/index.d.ts
-rw-r--r--  0 0      0        2288 Oct 26  1985 package/config/fs.d.ts
-rw-r--r--  0 0      0         140 Oct 26  1985 package/hooks/useActiveMonacoEditor.d.ts
-rw-r--r--  0 0      0         246 Oct 26  1985 package/hooks/useCommand.d.ts
-rw-r--r--  0 0      0          70 Oct 26  1985 package/hooks/useDataURL.d.ts
-rw-r--r--  0 0      0         139 Oct 26  1985 package/hooks/useDiscardChanges.d.ts
-rw-r--r--  0 0      0         207 Oct 26  1985 package/hooks/useMonacoAction.d.ts
-rw-r--r--  0 0      0          55 Oct 26  1985 package/hooks/usePrevious.d.ts
-rw-r--r--  0 0      0        2078 Oct 26  1985 package/hooks/useStore.d.ts
-rw-r--r--  0 0      0         157 Oct 26  1985 package/stores/__mocks__/studio.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/__tests__/git.unit.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/__tests__/jsonSchema.spec.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/__tests__/tag.spec.d.ts
-rw-r--r--  0 0      0         810 Oct 26  1985 package/stores/apiServices.d.ts
-rw-r--r--  0 0      0         267 Oct 26  1985 package/stores/app/apollo.d.ts
-rw-r--r--  0 0      0         891 Oct 26  1985 package/stores/app/cache.d.ts
-rw-r--r--  0 0      0        1316 Oct 26  1985 package/stores/app/index.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/cache/__tests__/index.unit.d.ts
-rw-r--r--  0 0      0         637 Oct 26  1985 package/stores/cache/idbStorage.d.ts
-rw-r--r--  0 0      0        1739 Oct 26  1985 package/stores/cache/index.d.ts
-rw-r--r--  0 0      0         256 Oct 26  1985 package/stores/cache/memory.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/command/__tests__/command.unit.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/command/__tests__/registry.unit.d.ts
-rw-r--r--  0 0      0         972 Oct 26  1985 package/stores/command/command.d.ts
-rw-r--r--  0 0      0         951 Oct 26  1985 package/stores/command/index.d.ts
-rw-r--r--  0 0      0         973 Oct 26  1985 package/stores/command/registry.d.ts
-rw-r--r--  0 0      0        1386 Oct 26  1985 package/stores/command/types.d.ts
-rw-r--r--  0 0      0         332 Oct 26  1985 package/stores/dirtyFiles.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/__tests__/editor.int.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/__tests__/handleChangeCursorPosition.unit.d.ts
-rw-r--r--  0 0      0        3020 Oct 26  1985 package/stores/editor/editor.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/formatters/markdown.d.ts
-rw-r--r--  0 0      0         633 Oct 26  1985 package/stores/editor/handleChangeCursorPosition.d.ts
-rw-r--r--  0 0      0         164 Oct 26  1985 package/stores/editor/hooks.d.ts
-rw-r--r--  0 0      0         455 Oct 26  1985 package/stores/editor/index.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/markdown-plugin/__tests__/commands.spec.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/markdown-plugin/__tests__/tables.spec.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/markdown-plugin/__tests__/transformers.spec.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/editor/markdown-plugin/__tests__/utils.spec.d.ts
-rw-r--r--  0 0      0        1146 Oct 26  1985 package/stores/editor/markdown-plugin/commands.d.ts
-rw-r--r--  0 0      0         419 Oct 26  1985 package/stores/editor/markdown-plugin/consts.d.ts
-rw-r--r--  0 0      0         263 Oct 26  1985 package/stores/editor/markdown-plugin/menus.d.ts
-rw-r--r--  0 0      0         433 Oct 26  1985 package/stores/editor/markdown-plugin/tables.d.ts
-rw-r--r--  0 0      0        1488 Oct 26  1985 package/stores/editor/markdown-plugin/transformers.d.ts
-rw-r--r--  0 0      0          59 Oct 26  1985 package/stores/editor/markdown-plugin/types.d.ts
-rw-r--r--  0 0      0        1738 Oct 26  1985 package/stores/editor/markdown-plugin/utils.d.ts
-rw-r--r--  0 0      0         377 Oct 26  1985 package/stores/editor/parser.d.ts
-rw-r--r--  0 0      0         610 Oct 26  1985 package/stores/editor/types.d.ts
-rw-r--r--  0 0      0         842 Oct 26  1985 package/stores/file/index.d.ts
-rw-r--r--  0 0      0         935 Oct 26  1985 package/stores/fileTree.d.ts
-rw-r--r--  0 0      0         807 Oct 26  1985 package/stores/git/credentialManager.d.ts
-rw-r--r--  0 0      0        3618 Oct 26  1985 package/stores/git/git.d.ts
-rw-r--r--  0 0      0          24 Oct 26  1985 package/stores/git/index.d.ts
-rw-r--r--  0 0      0        2269 Oct 26  1985 package/stores/graph.d.ts
-rw-r--r--  0 0      0         574 Oct 26  1985 package/stores/index.d.ts
-rw-r--r--  0 0      0        1076 Oct 26  1985 package/stores/jsonSchema.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/menu/__tests__/index.unit.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/stores/menu/__tests__/menu.unit.d.ts
-rw-r--r--  0 0      0         463 Oct 26  1985 package/stores/menu/index.d.ts
-rw-r--r--  0 0      0        8393 Oct 26  1985 package/stores/menu/menu.d.ts
-rw-r--r--  0 0      0        1518 Oct 26  1985 package/stores/menu/types.d.ts
-rw-r--r--  0 0      0         684 Oct 26  1985 package/stores/nodes/article.d.ts
-rw-r--r--  0 0      0        1636 Oct 26  1985 package/stores/nodes/create.d.ts
-rw-r--r--  0 0      0        1018 Oct 26  1985 package/stores/nodes/httpOperation.d.ts
-rw-r--r--  0 0      0         758 Oct 26  1985 package/stores/nodes/httpService.d.ts
-rw-r--r--  0 0      0         173 Oct 26  1985 package/stores/nodes/index.d.ts
-rw-r--r--  0 0      0         823 Oct 26  1985 package/stores/nodes/model.d.ts
-rw-r--r--  0 0      0         648 Oct 26  1985 package/stores/nodes/types.d.ts
-rw-r--r--  0 0      0         767 Oct 26  1985 package/stores/notification.d.ts
-rw-r--r--  0 0      0         150 Oct 26  1985 package/stores/omnibar.d.ts
-rw-r--r--  0 0      0        4225 Oct 26  1985 package/stores/plugin/api.d.ts
-rw-r--r--  0 0      0         662 Oct 26  1985 package/stores/prism.d.ts
-rw-r--r--  0 0      0         566 Oct 26  1985 package/stores/project/create.d.ts
-rw-r--r--  0 0      0         165 Oct 26  1985 package/stores/project/index.d.ts
-rw-r--r--  0 0      0        1258 Oct 26  1985 package/stores/project/registry.d.ts
-rw-r--r--  0 0      0         586 Oct 26  1985 package/stores/project/store.d.ts
-rw-r--r--  0 0      0         555 Oct 26  1985 package/stores/search.d.ts
-rw-r--r--  0 0      0         835 Oct 26  1985 package/stores/settings.d.ts
-rw-r--r--  0 0      0        1072 Oct 26  1985 package/stores/studio/cache.d.ts
-rw-r--r--  0 0      0        2816 Oct 26  1985 package/stores/studio/index.d.ts
-rw-r--r--  0 0      0        2134 Oct 26  1985 package/stores/tag.d.ts
-rw-r--r--  0 0      0        2412 Oct 26  1985 package/stores/ui.d.ts
-rw-r--r--  0 0      0    10247544 Oct 26  1985 package/studio/worker.web.js
-rw-r--r--  0 0      0         361 Oct 26  1985 package/utils/createCollection.d.ts
-rw-r--r--  0 0      0         960 Oct 26  1985 package/utils/createRegistry.d.ts
-rw-r--r--  0 0      0         125 Oct 26  1985 package/utils/getNormalizedType.d.ts
-rw-r--r--  0 0      0         212 Oct 26  1985 package/utils/getProvider.d.ts
-rw-r--r--  0 0      0         713 Oct 26  1985 package/utils/graph.d.ts
-rw-r--r--  0 0      0        2144 Oct 26  1985 package/utils/http.d.ts
-rw-r--r--  0 0      0          66 Oct 26  1985 package/utils/json.d.ts
-rw-r--r--  0 0      0         240 Oct 26  1985 package/utils/nameProviders.d.ts
-rw-r--r--  0 0      0         187 Oct 26  1985 package/utils/object.d.ts
-rw-r--r--  0 0      0          63 Oct 26  1985 package/utils/openUrlInBrowser.d.ts
-rw-r--r--  0 0      0         131 Oct 26  1985 package/utils/pendo.d.ts
-rw-r--r--  0 0      0         212 Oct 26  1985 package/utils/strings.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/utils/tree-transformers/__tests__/assignId.spec.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/utils/tree-transformers/__tests__/diagnostics.unit.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/utils/tree-transformers/__tests__/toTreeListNode.unit.d.ts
-rw-r--r--  0 0      0         135 Oct 26  1985 package/utils/tree-transformers/assignId.d.ts
-rw-r--r--  0 0      0         403 Oct 26  1985 package/utils/tree-transformers/createObservableTreeListNode.d.ts
-rw-r--r--  0 0      0         197 Oct 26  1985 package/utils/tree-transformers/diagnostics.d.ts
-rw-r--r--  0 0      0         145 Oct 26  1985 package/utils/tree-transformers/index.d.ts
-rw-r--r--  0 0      0         496 Oct 26  1985 package/utils/tree-transformers/toTreeListNode.d.ts
-rw-r--r--  0 0      0         249 Oct 26  1985 package/utils/versionProviders.d.ts
-rw-r--r--  0 0      0         315 Oct 26  1985 package/worker/client.d.ts
-rw-r--r--  0 0      0         524 Oct 26  1985 package/worker/file/index.d.ts
-rw-r--r--  0 0      0         277 Oct 26  1985 package/worker/file/types.d.ts
-rw-r--r--  0 0      0         465 Oct 26  1985 package/worker/fsutils.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/worker/git/__tests__/worker.unit.d.ts
-rw-r--r--  0 0      0        1517 Oct 26  1985 package/worker/git/index.d.ts
-rw-r--r--  0 0      0        3181 Oct 26  1985 package/worker/git/types.d.ts
-rw-r--r--  0 0      0         901 Oct 26  1985 package/worker/graph/index.d.ts
-rw-r--r--  0 0      0        1396 Oct 26  1985 package/worker/graph/types.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/worker/magic-portal/__tests__/magic-portal.unit.d.ts
-rw-r--r--  0 0      0         466 Oct 26  1985 package/worker/magic-portal/createPortalPair.d.ts
-rw-r--r--  0 0      0          33 Oct 26  1985 package/worker/magic-portal/index.d.ts
-rw-r--r--  0 0      0        1903 Oct 26  1985 package/worker/magic-portal/magic-portal.d.ts
-rw-r--r--  0 0      0          12 Oct 26  1985 package/worker/worker.d.ts
-rwxr-xr-x  0 0      0         132 Oct 26  1985 package/node_modules/@stoplight/spectral/bin/run
-rw-r--r--  0 0      0          31 Oct 26  1985 package/node_modules/@stoplight/spectral/bin/run.cmd
-rw-r--r--  0 0      0        1047 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/commands/lint.d.ts
-rw-r--r--  0 0      0        8725 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/commands/lint.js
-rw-r--r--  0 0      0        6887 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/commands/lint.js.map
-rw-r--r--  0 0      0          38 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/index.d.ts
-rw-r--r--  0 0      0         182 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/index.js
-rw-r--r--  0 0      0         144 Oct 26  1985 package/node_modules/@stoplight/spectral/cli/index.js.map
-rw-r--r--  0 0      0         450 Oct 26  1985 package/node_modules/@stoplight/spectral/config/configLoader.d.ts
-rw-r--r--  0 0      0        2009 Oct 26  1985 package/node_modules/@stoplight/spectral/config/configLoader.js
-rw-r--r--  0 0      0        1817 Oct 26  1985 package/node_modules/@stoplight/spectral/config/configLoader.js.map
-rw-r--r--  0 0      0          51 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/index.d.ts
-rw-r--r--  0 0      0         247 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/index.js
-rw-r--r--  0 0      0         143 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/index.js.map
-rw-r--r--  0 0      0         103 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/json.d.ts
-rw-r--r--  0 0      0         484 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/json.js
-rw-r--r--  0 0      0         520 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/json.js.map
-rw-r--r--  0 0      0         106 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/stylish.d.ts
-rw-r--r--  0 0      0        3545 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/stylish.js
-rw-r--r--  0 0      0        3670 Oct 26  1985 package/node_modules/@stoplight/spectral/formatters/stylish.js.map
-rw-r--r--  0 0      0         144 Oct 26  1985 package/node_modules/@stoplight/spectral/fs/reader.d.ts
-rw-r--r--  0 0      0        1178 Oct 26  1985 package/node_modules/@stoplight/spectral/fs/reader.js
-rw-r--r--  0 0      0         861 Oct 26  1985 package/node_modules/@stoplight/spectral/fs/reader.js.map
-rw-r--r--  0 0      0         122 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/alphabetical.d.ts
-rw-r--r--  0 0      0        1114 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/alphabetical.js
-rw-r--r--  0 0      0        1130 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/alphabetical.js.map
-rw-r--r--  0 0      0         119 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/enumeration.d.ts
-rw-r--r--  0 0      0         433 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/enumeration.js
-rw-r--r--  0 0      0         469 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/enumeration.js.map
-rw-r--r--  0 0      0         186 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/index.d.ts
-rw-r--r--  0 0      0         495 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/index.js
-rw-r--r--  0 0      0         517 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/index.js.map
-rw-r--r--  0 0      0         118 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/length.d.ts
-rw-r--r--  0 0      0         991 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/length.js
-rw-r--r--  0 0      0        1033 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/length.js.map
-rw-r--r--  0 0      0         121 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/pattern.d.ts
-rw-r--r--  0 0      0        1078 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/pattern.js
-rw-r--r--  0 0      0        1112 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/pattern.js.map
-rw-r--r--  0 0      0         122 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema-path.d.ts
-rw-r--r--  0 0      0        1106 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema-path.js
-rw-r--r--  0 0      0        1158 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema-path.js.map
-rw-r--r--  0 0      0         110 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema.d.ts
-rw-r--r--  0 0      0        3142 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema.js
-rw-r--r--  0 0      0        2951 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/schema.js.map
-rw-r--r--  0 0      0          78 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/truthy.d.ts
-rw-r--r--  0 0      0         343 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/truthy.js
-rw-r--r--  0 0      0         374 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/truthy.js.map
-rw-r--r--  0 0      0         109 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/xor.d.ts
-rw-r--r--  0 0      0         591 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/xor.js
-rw-r--r--  0 0      0         669 Oct 26  1985 package/node_modules/@stoplight/spectral/functions/xor.js.map
-rw-r--r--  0 0      0          28 Oct 26  1985 package/node_modules/@stoplight/spectral/index.d.ts
-rw-r--r--  0 0      0         198 Oct 26  1985 package/node_modules/@stoplight/spectral/index.js
-rw-r--r--  0 0      0         116 Oct 26  1985 package/node_modules/@stoplight/spectral/index.js.map
-rw-r--r--  0 0      0       10760 Oct 26  1985 package/node_modules/@stoplight/spectral/LICENSE
-rw-r--r--  0 0      0         430 Oct 26  1985 package/node_modules/@stoplight/spectral/linter.d.ts
-rw-r--r--  0 0      0        4984 Oct 26  1985 package/node_modules/@stoplight/spectral/linter.js
-rw-r--r--  0 0      0        4613 Oct 26  1985 package/node_modules/@stoplight/spectral/linter.js.map
-rw-r--r--  0 0      0        2366 Oct 26  1985 package/node_modules/@stoplight/spectral/meta/rule.schema.json
-rw-r--r--  0 0      0         498 Oct 26  1985 package/node_modules/@stoplight/spectral/meta/ruleset.schema.json
-rw-r--r--  0 0      0        1348 Oct 26  1985 package/node_modules/@stoplight/spectral/package.json
-rw-r--r--  0 0      0        7804 Oct 26  1985 package/node_modules/@stoplight/spectral/README.md
-rw-r--r--  0 0      0         102 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/ajv.d.ts
-rw-r--r--  0 0      0         348 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/ajv.js
-rw-r--r--  0 0      0         384 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/ajv.js.map
-rw-r--r--  0 0      0          96 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/index.d.ts
-rw-r--r--  0 0      0         515 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/index.js
-rw-r--r--  0 0      0         431 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/index.js.map
-rw-r--r--  0 0      0         250 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/message.d.ts
-rw-r--r--  0 0      0         501 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/message.js
-rw-r--r--  0 0      0         664 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/message.js.map
-rw-r--r--  0 0      0         109 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOp2xxResponse/index.d.ts
-rw-r--r--  0 0      0         463 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOp2xxResponse/index.js
-rw-r--r--  0 0      0         532 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOp2xxResponse/index.js.map
-rw-r--r--  0 0      0         118 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpFormDataConsumeCheck/index.d.ts
-rw-r--r--  0 0      0         656 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpFormDataConsumeCheck/index.js
-rw-r--r--  0 0      0         616 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpFormDataConsumeCheck/index.js.map
-rw-r--r--  0 0      0         106 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpIdUnique/index.d.ts
-rw-r--r--  0 0      0        1122 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpIdUnique/index.js
-rw-r--r--  0 0      0        1097 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpIdUnique/index.js.map
-rw-r--r--  0 0      0         104 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpParams/index.d.ts
-rw-r--r--  0 0      0        5124 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpParams/index.js
-rw-r--r--  0 0      0        4020 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpParams/index.js.map
-rw-r--r--  0 0      0         237 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpSecurityDefined/index.d.ts
-rw-r--r--  0 0      0        1305 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpSecurityDefined/index.js
-rw-r--r--  0 0      0        1203 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasOpSecurityDefined/index.js.map
-rw-r--r--  0 0      0         105 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasPathParam/index.d.ts
-rw-r--r--  0 0      0        5239 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasPathParam/index.js
-rw-r--r--  0 0      0        4410 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/functions/oasPathParam/index.js.map
-rw-r--r--  0 0      0         294 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/index.d.ts
-rw-r--r--  0 0      0        1165 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/index.js
-rw-r--r--  0 0      0         656 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/index.js.map
-rw-r--r--  0 0      0       11304 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas/ruleset.json
-rw-r--r--  0 0      0         174 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas2/index.d.ts
-rw-r--r--  0 0      0         524 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas2/index.js
-rw-r--r--  0 0      0         336 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas2/index.js.map
-rw-r--r--  0 0      0       59594 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas2/ruleset.json
-rw-r--r--  0 0      0         174 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas3/index.d.ts
-rw-r--r--  0 0      0         524 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas3/index.js
-rw-r--r--  0 0      0         336 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas3/index.js.map
-rw-r--r--  0 0      0       75056 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/oas3/ruleset.json
-rw-r--r--  0 0      0          68 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/path.d.ts
-rw-r--r--  0 0      0         407 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/path.js
-rw-r--r--  0 0      0         453 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/path.js.map
-rw-r--r--  0 0      0         507 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/reader.d.ts
-rw-r--r--  0 0      0        1163 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/reader.js
-rw-r--r--  0 0      0        1003 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/reader.js.map
-rw-r--r--  0 0      0         108 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/validation.d.ts
-rw-r--r--  0 0      0         559 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/validation.js
-rw-r--r--  0 0      0         519 Oct 26  1985 package/node_modules/@stoplight/spectral/rulesets/validation.js.map
-rw-r--r--  0 0      0         325 Oct 26  1985 package/node_modules/@stoplight/spectral/runner.d.ts
-rw-r--r--  0 0      0        2261 Oct 26  1985 package/node_modules/@stoplight/spectral/runner.js
-rw-r--r--  0 0      0        2232 Oct 26  1985 package/node_modules/@stoplight/spectral/runner.js.map
-rw-r--r--  0 0      0         798 Oct 26  1985 package/node_modules/@stoplight/spectral/spectral.d.ts
-rw-r--r--  0 0      0        4335 Oct 26  1985 package/node_modules/@stoplight/spectral/spectral.js
-rw-r--r--  0 0      0        3670 Oct 26  1985 package/node_modules/@stoplight/spectral/spectral.js.map
-rw-r--r--  0 0      0         408 Oct 26  1985 package/node_modules/@stoplight/spectral/types/config.d.ts
-rw-r--r--  0 0      0         462 Oct 26  1985 package/node_modules/@stoplight/spectral/types/config.js
-rw-r--r--  0 0      0         322 Oct 26  1985 package/node_modules/@stoplight/spectral/types/config.js.map
-rw-r--r--  0 0      0         323 Oct 26  1985 package/node_modules/@stoplight/spectral/types/enums.d.ts
-rw-r--r--  0 0      0         731 Oct 26  1985 package/node_modules/@stoplight/spectral/types/enums.js
-rw-r--r--  0 0      0         447 Oct 26  1985 package/node_modules/@stoplight/spectral/types/enums.js.map
-rw-r--r--  0 0      0         450 Oct 26  1985 package/node_modules/@stoplight/spectral/types/function.d.ts
-rw-r--r--  0 0      0         113 Oct 26  1985 package/node_modules/@stoplight/spectral/types/function.js
-rw-r--r--  0 0      0         117 Oct 26  1985 package/node_modules/@stoplight/spectral/types/function.js.map
-rw-r--r--  0 0      0         105 Oct 26  1985 package/node_modules/@stoplight/spectral/types/index.d.ts
-rw-r--r--  0 0      0         195 Oct 26  1985 package/node_modules/@stoplight/spectral/types/index.js
-rw-r--r--  0 0      0         125 Oct 26  1985 package/node_modules/@stoplight/spectral/types/index.js.map
-rw-r--r--  0 0      0        1796 Oct 26  1985 package/node_modules/@stoplight/spectral/types/rule.d.ts
-rw-r--r--  0 0      0         109 Oct 26  1985 package/node_modules/@stoplight/spectral/types/rule.js
-rw-r--r--  0 0      0         109 Oct 26  1985 package/node_modules/@stoplight/spectral/types/rule.js.map
-rw-r--r--  0 0      0         187 Oct 26  1985 package/node_modules/@stoplight/spectral/types/ruleset.d.ts
-rw-r--r--  0 0      0         112 Oct 26  1985 package/node_modules/@stoplight/spectral/types/ruleset.js
-rw-r--r--  0 0      0         115 Oct 26  1985 package/node_modules/@stoplight/spectral/types/ruleset.js.map
-rw-r--r--  0 0      0        1097 Oct 26  1985 package/node_modules/@stoplight/spectral/types/spectral.d.ts
-rw-r--r--  0 0      0         113 Oct 26  1985 package/node_modules/@stoplight/spectral/types/spectral.js
-rw-r--r--  0 0      0         117 Oct 26  1985 package/node_modules/@stoplight/spectral/types/spectral.js.map
-rw-r--r--  0 0      0          32 Oct 26  1985 package/node_modules/@stoplight/spectral/yalc.sig
</pre></code>
</details>